anndata/_core/sparse_dataset.py

```diff
@@ -340,10 +360,62 @@ def value(self) -> ss.spmatrix:
     def __repr__(self) -> str:
         return f"{type(self).__name__}: backend {self.backend}, shape {self.shape}, data_dtype {self.dtype}"
 
+    @property
+    def _efficient_axis(self):
+        """The axis over which the sparse matrix is efficient (i.e., row for csr)"""
+        formats = ["csr", "csc"]
+        return formats.index(self.format)
+
+    def _efficient_index(self, indices):
```

Comment:
Maybe a better thing to call this?  This is not my forte.  Would be great to chat IRL

Reply:
"compressed axis" I think the more common term

Reply:
Oh that is so much better, yes.